Client/Frontend/Widgets/AlternateSimpleHighlightCell.swift

```diff
@@ -137,32 +137,19 @@ class AlternateSimpleHighlightCell: UITableViewCell {
         fatalError("init(coder:) has not been implemented")
     }
 
-    func setImageWithURL(_ url: URL) {
-        siteImageView.sd_setImage(with: url) { (img, err, type, url) -> Void in
-            guard let img = img else {
-                return
-            }
-            // Resize an Image to a specfic size to make sure that it doesnt appear bigger than it needs to (32px) inside a larger frame (48px).
-            self.siteImageView.image = img.createScaled(CGSize(width: 32, height: 32))
-            self.siteImageView.image?.getColors(scaleDownSize: CGSize(width: 25, height: 25)) { colors in
-                self.siteImageView.backgroundColor = colors.backgroundColor ?? UIColor.lightGray
-            }
-        }
-    }
-
     override func prepareForReuse() {
         super.prepareForReuse()
         self.siteImageView.image = nil
+        contentView.backgroundColor = UIColor.clear
+        siteImageView.backgroundColor = UIColor.clear
     }
 
     func configureWithSite(_ site: Site) {
-        if let icon = site.icon, let url = URL(string:icon.url) {
-            self.setImageWithURL(url)
-        } else {
-            let url = site.url.asURL!
-            self.siteImageView.image = FaviconFetcher.getDefaultFavicon(url)
-            self.siteImageView.backgroundColor = FaviconFetcher.getDefaultColor(url)
-        }
+        self.siteImageView.setIcon(site.icon, forURL: URL(string: (site.icon?.url ?? site.url)), completed: { (color, url) in
```

Comment:
You can use the method that lets you pass the site in. That should work instead of you having to use this method.